NAMESPACE

```diff
@@ -63,8 +63,11 @@ export(use_tidy_contributing)
 export(use_tidy_description)
 export(use_tidy_eval)
 export(use_tidy_issue_template)
+export(use_tidy_style)
 export(use_tidy_support)
 export(use_tidy_versions)
 export(use_travis)
 export(use_usethis)
 export(use_vignette)
+importFrom(styler,style_pkg)
+importFrom(styler,tidyverse_style)
```

Comment:
We generally don't do this.

Reply:
So should I just use `::` whenever I need an external function?

Reply:
Yes.